command/loadbot/params.go

```diff
@@ -187,5 +192,49 @@ func (p *loadbotParams) generateConfig(
 		GasPrice:         p.gasPrice,
 		GasLimit:         p.gasLimit,
 		ContractArtifact: p.contractArtifact,
+		ConstructorArgs:  p.constructorArgs,
+		MaxWait:          p.maxWait,
 	}
 }
+
+func (p *loadbotParams) initContractArtifactAndArgs() error {
+	var (
+		ctrArtifact *generator.ContractArtifact
+		ctrArgs     []byte
+		err         error
+	)
+
+	switch p.mode {
+	case erc20:
+		ctrArtifact = &generator.ContractArtifact{
+			Bytecode: ERC20BIN,
+			ABI:      abi.MustNewABI(ERC20ABI),
+		}
+
+		if ctrArgs, err = abi.Encode(
+			[]string{"4314500000", "ZexCoin", "ZEX"}, ctrArtifact.ABI.Constructor.Inputs); err != nil {
```

Comment:
What does `4314500000` mean? Is there a possibility that you used all tokens during transfer?

Reply:
These are totally arbitrary values.
The ```4314500000``` is the total erc20 token supply.
It's a 5 decimal token, so the value is actually ```43145.00000```.
There is a possibility to use all tokens if someone sets ```--count```  to a high value, like ```100000``` for example.

Should we change this or a higher value or make it configurable maybe?

Reply:
I'd have the hardcoded values moved into their own (global) variables, so they can be configured from there.

Reply:
Moved constructor args to const 168f73cceef7ce3e718b2c001c66af17ecd8acb9

Reply:
I think they should be configurable. `4314500000` looks random value. You need to write the reason why `4314500000` is proper as a comment around constants at least.

Reply:
Don't think that anyone will want to configure this, so I think we don't need to add unnecessary flags. 
I'd say that there just has to be enough funds to run all the transactions the user defined.

I've set `5000000000` as total token supply, set the transfer value to `0.001` and I've added comments.
This should be more than enough for any loadbot scenario.

@Kourin1996 ac607d15a6adf365a17996b8a84e98fa5513dad6